__tests__/utils/log.spec.js

```diff
@@ -61,7 +61,7 @@ describe('log functions', () => {
     it('should do nothing if called with an index out of range', () => {
```

Comment:
Next two test above this should now call all 6 steps.
tbh i was never happy with them, perhaps they could instead loop over the number of steps in the steps array so when we add steps they update automatically?

Reply:
good point. Ive added something rough but will get us going.